lib/convection/model/template/resource/aws_efs_file_system.rb

```diff
@@ -0,0 +1,24 @@
+require_relative '../resource'
+
+module Convection
+  module Model
+    class Template
+      class Resource
+        # Represents an {https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-efs-filesystem.html
+        # Amazon EFS File System}
+        class EFSFileSystem < Resource
+          include Model::Mixin::Taggable
```

Comment:
@rdickey-r7, good idea! I added a `file_system_tag` alias in https://github.com/rapid7/convection/pull/259/commits/8fe117c38c90b84137ac26ed0d46da3ca32de971.